ts/pages/staking/wizard/wizard.tsx

```diff
@@ -132,7 +122,28 @@ export const StakingWizard: React.FC<StakingWizardProps> = props => {
 
         // tslint:disable-next-line:no-floating-promises
         fetchAndSetStakingStats();
-    }, [networkId, apiClient]);
+    }, [chainId, apiClient]);
+
+    useEffect(() => {
+        const loadBalances = async () => {
+            // tslint:disable-next-line:no-shadowed-variable
+            const { zrxBalanceBaseUnitAmount } = await asyncDispatcher.fetchAccountBalanceAndDispatchToStoreAsync(
+                account,
+                connector,
+                dispatcher,
+                chainId,
+            );
+
+            // tslint:disable-next-line:no-shadowed-variable
+            const zrxBalance = Web3Wrapper.toUnitAmount(zrxBalanceBaseUnitAmount, constants.DECIMAL_PLACES_ZRX);
+
+            setZrxBalance(zrxBalance);
+            setZrxBalanceBaseUnitAmount(zrxBalanceBaseUnitAmount);
+        };
+
+        // tslint:disable-next-line:no-floating-promises
+        loadBalances();
+    }, [account, chainId, connector]);
```

Comment:
Ah yeah, looks like you're loading it here. So just need to update where you're getting the balance from.

Reply:
I don't think I follow, can you please explain?

Reply:
Oh nothing actionable here. 

Let's just make sure we fix the `zrxAllowanceBaseUnitAmount` on the comment(s) above.

Reply:
I just added the allowance data also, I think its resolved